Dispatcher/UI/Screens/SplashPage/SplashViewController.swift

```diff
@@ -10,8 +13,8 @@ class SplashViewController: UIViewController {
         super.viewDidAppear(animated)
 
         DispatchQueue.main.asyncAfter(deadline: .now() + 3.0) {
```

Comment:
alot of things are going on here...
this can be a function with an easy description of what this block does
the "identifiers" should be local in the scope not static in class